Placeholders in Classic (#4884)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/archive-sign-in-to-spa-embedded-widget/main/angular/guide.md

| Login redirect URIs  | `http://localhost:4200/login/callback`              |
| Logout redirect URIs | `http://localhost:4200/login`                       |
| Allowed grant types  | Authorization Code                                  |

`{clientId}` placeholders further in this tutorial should be replaced by the client ID of the created application.

> > **Note:** It is important to choose the appropriate application type for apps which are public clients. Failing to do so may result in Okta API endpoints attempting to verify an app's client secret, which public clients are not designed to have, hence breaking the sign-in or sign-out flow.

Reviewer: ```suggestion
> **Note:** It's important to choose the appropriate application type for public client apps. Failing to do so may result in Okta API endpoints attempting to verify an app's client secret, which public clients are not designed to have, hence breaking the sign-in or sign-out flow.
```
